client/src/pages/locations/GeoLocation.js

```diff
@@ -0,0 +1,122 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Button, Col, ControlLabel, FormGroup } from "react-bootstrap"
+import { Field } from "formik"
+import { Location } from "models"
+import REMOVE_ICON from "resources/delete.png"
+import * as FieldHelper from "components/FieldHelper"
+
+export const GEO_LOCATION_DISPLAY_TYPE = {
+  FORM_FIELD: "FORM_FIELD",
+  GENERIC: "GENERIC"
+}
+
+const GeoLocation = ({
+  lat,
+  lng,
+  setFieldTouched,
+  setFieldValue,
+  isSubmitting,
+  editable,
+  displayType
+}) => {
+  const setTouched = touched => {
+    setFieldTouched("lat", touched, false)
+    setFieldTouched("lng", touched, false)
```

Comment:
`setFieldTouched` is called unconditionally, yet according to the propTypes it is not required, and there's no default prop value supplied.

Reply:
Added custom propType validation for `setFieldTouched`.